scripts/dts/edtlib.py

```diff
@@ -724,9 +724,19 @@ def unit_addr(self):
 
         addr = _translate(addr, self._node)
 
-        if self.regs and self.regs[0].addr != addr:
-            self.edt._warn("unit-address and first reg (0x{:x}) don't match "
-                           "for {}".format(self.regs[0].addr, self.name))
+        try:
+            EXTRA_DTC_FLAGS = os.environ.get("EXTRA_DTC_FLAGS")
```

Comment:
We should not use env vars to communicate information between the build system and underlying scripts. CLI arguments should be used.